compiler/rustc_mir_build/src/errors.rs

```diff
@@ -33,6 +33,11 @@ pub(crate) struct CallToDeprecatedSafeFnRequiresUnsafe {
 #[derive(Subdiagnostic)]
 #[multipart_suggestion(mir_build_suggestion, applicability = "machine-applicable")]
 pub(crate) struct CallToDeprecatedSafeFnRequiresUnsafeSub {
+    pub(crate) indent: String,
+    #[suggestion_part(
+        code = "{indent}// FIXME: Audit that the environment access only happens in single-threaded code.\n"
```

Comment:
Is there any precedent for inserting "TODO"/"FIXME" comments as part of suggestions? It might be more popular among downstream Rust users to put TODO comments instead of FIXME compared to our codebase where TODO should not exist.

Reply:
My quick grepping around in the code base did not find any TODO/FIXME comments as part of suggestions.

Reply:
Do you have a preference of what to use?

Reply:
I would prefer TODO

Reply:
Done.